Makefile

```diff
@@ -0,0 +1,43 @@
+GO_EXEC := $(shell which go)
```

Comment:
This can use some cleanup or even removal in favouring of doing this with `yarn`.

Reply:
umm this is to find the location of the local installation of go - why yarn ?

Reply:
the only thing I can think of is to make it generic is to use https://hub.docker.com/_/golang via docker so you need to have docker available instead of a golang stack.